File: week01/pset1/02-mario/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void print_row(int spaces, int bricks);
void print_col(int height);

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    print_col(height);
}

void print_row(int spaces, int bricks)
{
    for (int i = 0; i < spaces; i++)
    {
        printf(" ");
    }

    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }

    printf("  ");

    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }
    printf("\n");
}

void print_col(int height)
{

    for (int i = 0; i < height; i++)
    {
        int spaces = height - i - 1;
        print_row(spaces, i + 1);
    }
}

File: week06/pset6/sentimental-mario-less/mario.py
from cs50 import get_int


while True:
    n = get_int("Height: ")
    width = n + 1
    if (n > 0):
        break

for i in range(1, width):
    hashes = i + 1
    spaces = width - hashes

    print(" " * spaces, end="")
    print("#" * hashes)
